package.json

```diff
@@ -17,10 +17,10 @@
     "build:watch": "tsc && (tsc -w --preserveWatchOutput & nodemon)",
     "dev": "npm run build:watch",
     "start": "probot run ./lib/index.js",
-    "lint": "eslint src/**/*.ts",
-    "lint-fix": "eslint src/**/*.ts --fix",
-    "check-format": "prettier src/**/*.ts -l",
-    "format": "prettier src/**/*.ts --write",
+    "lint": "eslint \"src/**/*.ts\"",
```

Comment:
We usually use single quotes in these scripts if variable interpolation is not needed.